OpenRA.Mods.Common/Widgets/ScrollItemWidget.cs

```diff
@@ -57,6 +57,7 @@ public static ScrollItemWidget Setup(ScrollItemWidget template, Func<bool> isSel
 			w.IsVisible = () => true;
 			w.IsSelected = isSelected;
 			w.OnClick = onClick;
+			w.ItemKey = Convert.ToString(Game.CosmeticRandom.Next());
```

Comment:
That defeats the purpose of a hotkey (which is meant to be consistent), as it will be new each time.